default/bash/functions

```diff
@@ -25,14 +25,21 @@ format-drive() {
   else
     echo "WARNING: This will completely erase all data on $1 and label it '$2'."
     read -rp "Are you sure you want to continue? (y/N): " confirm
+
     if [[ "$confirm" =~ ^[Yy]$ ]]; then
       sudo wipefs -a "$1"
       sudo dd if=/dev/zero of="$1" bs=1M count=100 status=progress
       sudo parted -s "$1" mklabel gpt
-      sudo parted -s "$1" mkpart primary ext4 1MiB 100%
-      sudo mkfs.ext4 -L "$2" "$([[ $1 == *"nvme"* ]] && echo "${1}p1" || echo "${1}1")"
-      sudo chmod -R 777 "/run/media/$USER/$2"
-      echo "Drive $1 formatted and labeled '$2'."
+      sudo parted -s "$1" mkpart primary 1MiB 100%
+
+      partition="$([[ $1 == *"nvme"* ]] && echo "${1}p1" || echo "${1}1")"
+      sudo partprobe "$1" || true
+      sudo udevadm settle || true
+
+      omarchy-pkg-add exfatprogs
```

Comment:
Should `exfatprogs` be added to the list of required packages, rather than trying to install it when this function is run?

This will ensure that this function still works if the user has never run the function, and has no internet connection.